netflix/src/index.css

```diff
@@ -14,8 +15,55 @@ code {
 
 h1 {
   text-align: center;
+  color: #191414;
```

Comment:
Not a bug.

Instead of using RGB all across your CSS, you may want to at least declare CSS custom properties and re-use them:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties